docs/user/aws/install_upi.md

```diff
@@ -0,0 +1,432 @@
+
+```console
+$ tree
+   .
+   ├── auth
+   │   └── kubeconfig
+   ├── bootstrap.ign
+   ├── master.ign
+   ├── metadata.json
+   └── worker.ign
+```
+
+## Create/Identify the VPC to be Used
+
+You may create a VPC with various desirable characteristics for your situation (VPN, route tables, etc.). The recommended
+VPC configuration and a CloudFormation template is provided [here](../../../upi/aws/cloudformation/01_vpc.yaml).
+
+A created VPC via the template or manually should approximate a setup similar to this:
+
+![Example VPC Architecture Diagram](images/install_upi_vpc.png)
+
+## Create DNS entries and Load Balancers for Control Plane Components
+
+The recommended DNS and load balancer configuration within a CloudFormation template is provided
+[here](../../../upi/aws/cloudformation/02_cluster_infra.yaml). It uses a public hosted zone and creates a private hosted
+zone similar to the IPI installation method. It also creates load balancers and listeners the same way as the IPI
+installation method. This template can be run multiple times within a single VPC and in combination with the VPC
+template provided above.
+
+### Optional: Manually Create Load Balancer Configuration
+
+It is recommended to create a TCP load balancer for ports 6443 and 22623. The targets will be the master nodes.
+Port 6443 must be accessible to both clients external to the cluster and nodes within the cluster. Port 22623 must
+be accessible to nodes within the cluster.
+
+### Optional: Manually Create Route53 Hosted Zones & Records
+
+For the cluster name identified earlier in "Create Ignition Configs", you must create a DNS entry which resolves to
+your created load balancer. The entry api.$clustername.$domain should point to the load balancer.
+
+## Create Security Groups and IAM Roles
+
+The recommended security group and IAM configuration within a CloudFormation template is provided
+[here](../../../upi/aws/cloudformation/03_cluster_security.yaml). Run this template to get the minimal and permanent
+set of security groups and IAM roles needed for an operational cluster. It can also be inspected for the current
+set of required rules to facilitate manual creation.
+
+## Launch Temporary Bootstrap Resource
+
+The recommended bootstrap launch and other necessary, temporary security group plus IAM configuration and a CloudFormation
+template is provided [here](../../../upi/aws/cloudformation/04_cluster_bootstrap.yaml). Upload your generated bootstrap.ign
+file to an S3 bucket in your account and run this template to get a bootstrap node along with a predictable clean up of
+the resources when complete. It can also be inspected for the set of required attributes via manual creation.
+
+## Launch Permanent Master Nodes
+
+The recommended master launch and other necessary DNS entries for etcd are provided within a CloudFormation
+template [here](../../../upi/aws/cloudformation/05_cluster_master_nodes.yaml). Run this template to get a three master
```

Comment:
nit: "to get a three" -> "to get three"

Reply:
Also "master" -> "control-plane" (see also #1330)

Reply:
Fixed.